[FIX] Translation edit page breaks on update

ZoeS17 opened the pull request:
*FIX #36 
Make speaker and episode silently fail should their fields be left blank.

*FIX
Add translation link to the index blade

ZoeS17 committed:
[FIX] Translation edit page breaks on insert

*FIX
Make speaker and episode silently fail should their fields be left blank.

SKroell commented on app/Http/Controllers/WebController.php:
```diff
@@ -104,9 +104,9 @@ public function translationEditSubmit(Request $request, $id){
             'etymology' => $request->etymology,
             'leipzig' => $request->leipzig,
-            'episode' => $request->episode,
+            'episode' => isset($request->episode) ? $request->episode : '',
```
'episode' should default to 'other'

SKroell reviewed (changes requested):
We are still redoing a ton of the database stuff, so this is a good fix for now. 

Speaker should not be marked as NOT NULL and we will address that in the database revamp. But the 'episode' column should default to 'other' to correctly display it on the translations list.

ZoeS17 committed:
[Fix] Update dict_translations->episode to default to 'other'

*Fix
Update the episode field of dict_translations to default to 'other' instead of an empty string or NULL.

ZoeS17 reviewed (commented):
Is this what you were suggesting @SKroell?

SKroell commented:
> Is this what you were suggesting @SKroell?

Yes
